cmd/trigger-double-sign/main.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
put into /test if you want to keep it.

Reply:
how do you build into binary if put under test? i mean, will go_exectuable_build.sh build it and put into bin?